src/main.cpp

```diff
@@ -40,17 +44,16 @@ namespace YimMenu
 		ScriptMgr::AddScript(std::make_unique<Script>(&FeatureLoop));
 		ScriptMgr::AddScript(std::make_unique<Script>(&BlockControlsForUI));
 
-		g_HotkeySystem.RegisterCommands();
-
 		while (g_Running)
 		{
-			//Needed incase UI is malfunctioning or for emergencies
-			if (GetAsyncKeyState(VK_DELETE) & 0x8000)
+			// Needed incase UI is malfunctioning or for emergencies
+			if (GetAsyncKeyState(VK_DELETE) & 0x8000 && !NETWORK::NETWORK_IS_SESSION_STARTED())
```

Comment:
Atleast use the ptr if you insist unloading in online should absolutely be forbidden